infra.go

```diff
@@ -30,21 +30,21 @@ type InfrastructureResourceConnection struct {
 }
 
 type InfrastructureResourceSchemaInput struct {
-	Type string `json:"type"`
+	Type string `json:"type" yaml:"type"`
 }
 
 type InfrastructureResourceProviderInput struct {
-	AccountName  string `json:"accountName"`
-	ExternalURL  string `json:"externalUrl"`
-	ProviderName string `json:"providerName"`
+	AccountName  string `json:"accountName" yaml:"accountName"`
+	ExternalURL  string `json:"externalUrl" yaml:"externalUrl"`
+	ProviderName string `json:"providerName" yaml:"providerName"`
 }
 
 type InfrastructureResourceInput struct {
-	Type         *string                              `json:"providerResourceType,omitempty"`
+	Type         *string                              `json:"providerResourceType,omitempty" yaml:"providerResourceType"`
 	Schema       *InfrastructureResourceSchemaInput   `json:"schema,omitempty"`
-	ProviderData *InfrastructureResourceProviderInput `json:"providerData,omitempty"`
-	Owner        *ID                                  `json:"ownerId,omitempty"`
-	Data         JSON                                 `json:"data,omitempty" scalar:"true"`
+	ProviderData *InfrastructureResourceProviderInput `json:"providerData,omitempty" yaml:"providerData"`
+	Owner        *ID                                  `json:"ownerId,omitempty" yaml:"owner"`
+	Data         map[string]any                       `json:"data,omitempty" yaml:"data"`
```

Comment:
I noticed `InfrastructureResourceInput`'s `Data` struct tags no longer include `scalar:"true"` but `InfrastructureResource`'s `Data` struct tags above do include `scalar:"true"`. I'm guessing this is fine since there aren't any `graphql` tags here?

Reply:
Ya - just oversight - we'll be undoing this once the real fix is in